### PR TITLE
ASN.1 Module

### PR DESCRIPTION
**Pull request by seanturner**
Here's an ASN.1 module that I worked with Russ to craft.

Note that I included an updates header for 8018 because we're extending the module. Not hard over on including the header.

I am not very good with XML so hopefully I did not mess it up.

Addresses #6.

### CODE REVIEW

File: draft-ietf-lamps-pkcs12-pbmac1.xml [hunk 1]
```diff
@@ -221,6 +225,97 @@
...
+          <date month="February" year="2021"/>
+      </front>
+      <seriesInfo name="ITU-T Recommendation" value="X.690"/>
+      <seriesInfo name="ISO/IEC" value="8825-1:2021"/>
+      </reference>
+
+         <front>
+            <title>Information Technology - Abstract Syntax
+            Notation One (ASN.1):  Specification of basic notation.
+            ITU-T Recommendation X.680 (2002) | ISO/IEC 8824-1:2002.
```
Comment (tomato42) [inline draft-ietf-lamps-pkcs12-pbmac1.xml]: This is a duplicated `<front>` section or missing opening `<reference>`. I can fix it, but not sure which title you want to use...

**Comment by tomato42** [comment]
@seanturner I've fixed few other typos, just the issue with duplicated `<front>` remains

File: draft-ietf-lamps-pkcs12-pbmac1.xml (hunk 1 above)
Comment (seanturner) [reply draft-ietf-lamps-pkcs12-pbmac1.xml]: A bad cut and paste error. This can be deleted.

**Comment by seanturner** [comment]
The 2nd duplicate can be deleted. I also think there is a stray </reference> immediately after the dupe.

**Comment by tomato42** [comment]
@seanturner OK , removed. Now it compiles cleanly (there's just warning about too old date). Please check if all of my changes are ok, and I'll merge it after you give an-OK.

**Comment by seanturner** [comment]
@tomato42 I think that complaint is about line 56:

     <date day="6" month="November" year="2023" />

I think if you change it to the day you want to submit, it should go away. It did on my local copy when I ran xml2rfc.

**Comment by tomato42** [comment]
yes, it's just the date, I will change it when I submit new version

**Comment by tomato42** [comment]
Thanks!

**Pull request merged**